src/com/rcv/ResultsWriter.java

```diff
@@ -0,0 +1,228 @@
+package com.rcv;
+
+import org.apache.poi.ss.usermodel.Cell;
+import org.apache.poi.ss.usermodel.Row;
+import org.apache.poi.xssf.usermodel.XSSFSheet;
+import org.apache.poi.xssf.usermodel.XSSFWorkbook;
+
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.util.*;
+
+/**
+ * Created by Jon Moldover on 9/4/17.
+ *
+ * Helper class takes tabulation results data as input and generates results file
+ * Currently we support an xlsx spreadsheet which can be visualized in a web browser
+ *
+ */
+public class ResultsWriter {
+
+  // takes tabulation round tallies and generates a spreadsheet from them
+  public boolean generateSummarySpreadsheet(
+      int finalRound,
+      Map<Integer, Map<String, Integer>> roundTallies,
+      Map<String, Integer> candidatesToRoundEliminated,
+      String outputFilePath
+  ) {
+    XSSFWorkbook workbook = new XSSFWorkbook();
+    XSSFSheet worksheet = workbook.createSheet("Election Results");
+
+    // TODO: write the election header info here (contest name, total votes cast, threhold, etc..)
+
+    /////////////////////////////////////
+    // Round-by-round elections results
+    /////////////////////////////////////
+
+    // Row 0: Round headers
+    int rowCounter = 0;
+    org.apache.poi.ss.usermodel.Row headerRow = worksheet.createRow(rowCounter++);
+    for(int round = 1; round <= finalRound; round++) {
```

Comment:
nit: space after for

Reply:
I can't go there, apologies.  if you want to change it that is fine.

Reply:
Ha, OK. If you don't mind me fixing random things like this when I encounter them in the code, then I won't nitpick them in PRs.